resolwe/flow/models/data.py

```diff
@@ -486,3 +487,24 @@ class DataDependency(models.Model):
     parent = models.ForeignKey(Data, null=True, on_delete=models.SET_NULL, related_name='children_dependency')
     #: kind of dependency
     kind = models.CharField(max_length=16, choices=KIND_CHOICES)
+
+
+class DataLocation(models.Model):
+    """Location data of the data object."""
+
+    #: location (bound to data object's id)
+    location = models.CharField(max_length=30, unique=True)
```

Comment:
The notation `data.location.location` is a bit cumbersome. Could you name this differently? What exactly do you store in this field? Could we call it `slug` or `segment`?

Reply:
I suggest `path`, or maybe `subpath`, as in future there will be multiple storage types (local storage, s3,...) and this will be a subpath of the storage path.

Reply:
`subpath` it is.